front/pages/w/[wId]/builder/data-sources/[dsId]/index.tsx

```diff
@@ -109,6 +109,21 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
     );
     if (connectorRes.isOk()) {
       connector = connectorRes.value;
+    } else {
```

Comment:
We can remove the null `| null` above no?

Reply:
Nope, folders don't have a connectorId so no connector!